app/views/videos/show.html.slim

```diff
@@ -27,12 +27,25 @@
     /   a href="#"
     /     strong mostrar mais
   .watch-also
-    .grid-container
-      - unless @video.category.nil?
+    - unless @video.series.nil?
+      .text-center
+        = link_to "Voltar para a lista de episódios da série", @video.series, class: "button success"
+        .grid-container.text-left
+          h5
+            strong Assista também...
+          .owl-carousel
+            - @video.series.videos.each do |video|
+              .text-center
+                = link_to(video)
+                  = image_tag src="#{video.image_url}"
+                p = video.name
+    - unless @video.category.nil?
+      .grid-container
         h5
           strong Assista também...
         .owl-carousel
           - @video.category.videos.each do |video|
             .text-center
-              = image_tag href="http://lorempixel.com/311/175/animals"
+              = link_to(video)
+                = image_tag src="#{video.image_url}"
```

Comment:
Is this working? I've never seen this syntax. I think it is not :P 

`= image_tag video.image_url` should do the trick.

Reply:
I've seen you used this syntax all over the place, so I guess it works. Is it a Slim thing, mixing Ruby code with HTML attributes? Anyway, use the Ruby syntax whenever possible. It is cleaner and keeps us saner :P